nsxt/resource_nsxt_policy_project.go

```diff
@@ -212,8 +212,17 @@ func resourceNsxtPolicyProjectPatch(connector client.Connector, d *schema.Resour
 	if util.NsxVersionHigherOrEqual("9.1.0") {
 		// There should be just one object here
 		var spanReferences []model.SpanReference
-		defaultSpanPath := d.Get("default_span_path").(string)
-
+		defaultSpanPathinfce, isDefaultSet := d.GetOkExists("default_span_path")
```

Comment:
Sorry for being pedant, can you pick a variable name which is more readable than `defaultSpanPathinfce`?
Perhaps `defaultSpanPathFromSchema`?